data/Rajeshds.json

```diff
@@ -3,38 +3,53 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "An enthusiastic Tech Student. I develop softwares and websites, currently working on Full Stack Web Development.",
-  "avatar": "https://avatars.githubusercontent.com/u/104376917?s=400&u=932affed6d49d8afa5a12ebf2cf9b723f3db51eb&v=4",
+  "avatar": "https://www.github.com/rajeshds20.png",
```

Comment:
the avatar link is still wrong it should be `https://github.com/rajeshds20.png` without the `www`